### PR TITLE
PSR-1576 Clear out some data from the previous tax year - member paym…

### PR DESCRIPTION
**Pull request by cyeniceri**
…ents

### CODE REVIEW

File: test/prepop/MemberPrePopulationProcessorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+        |        "memberDetailsSectionCompleted": {
+        |          "0": {},
+        |          "1": {},
+        |          "2": {},
+        |          "3": {},
+        |          "4": {},
+        |          "5": {}
+        |        },
+        |        "memberStatus": {
+        |          "0": "Changed",
```
Comment (kberec) [inline test/prepop/MemberPrePopulationProcessorSpec.scala:115]: if the status is deleted, member should not be copied over.

Comment (cyeniceri) [reply test/prepop/MemberPrePopulationProcessorSpec.scala:115]: the records with `memberStatus : Deleted` was already marked as `"soft-deleted"` in baseReturn json in retrieval service and we aren't copying over these soft-deleted member records.

Comment (kberec) [reply test/prepop/MemberPrePopulationProcessorSpec.scala:115]: Do you need to test that soft-deleted members are not copied over?

Comment (cyeniceri) [reply test/prepop/MemberPrePopulationProcessorSpec.scala:115]: I put `"soft-deleted"` into baseReturn and verified that it's not within copied over userAnswer after cleanup.

**Review by kberec — APPROVED** [review]

**Review by kberec — APPROVED** [review]

**Pull request merged**